changelog.md

```diff
@@ -0,0 +1,7 @@
+- 1.12.0
+    Make upstream consumer group ID configurable
+- 1.12.1/2
+    Fix Route config validation
+- 1.12.3 (no tag yet)
+    * Update README with missing features
```

Comment:
wording confuses. they are not missing anymore.
update readme to document available/implemented configuration values and features